data/transactions/json_test.go

```diff
@@ -33,12 +33,11 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func decode(data string, v interface{}) error {
+func mustDecode(data string, v interface{}) {
```

Comment:
How about passing in `t`, so we can do a normal require.NoError() and see the problem, and the proper context?  Panics stop all concurrently running tests.

You would also use t.Helper(), probably.